app/classifier/index.cjsx

```diff
@@ -49,10 +75,15 @@ Classifier = React.createClass
       @loadSubject subject
     if nextProps.classification isnt @props.classification
       @prepareToClassify nextProps.classification
+    @context.geordi?.remember subjectID: @props.subject?.id
```

Comment:
Be careful about using `geordi.remember` with a corresponding `geordi.forget`. Also, I think this is overwriting the subjectID that was stored when the subject loaded, and I'm not sure if that's your intention. https://github.com/zooniverse/Panoptes-Front-End/blob/909f9b824f276febb57b0d6a24c012185432ef77/app/classifier/index.cjsx#L53